mercadopago/px-ios
# (Added)[PXN-3880] Added BIN setter in Tokenization services

## What has changed?

In our tokenization Services for Token creation, after a successful API call we were only setting the last four digits to our PXToken object in paymentData. 

With proposed change we also set the BIN (first six digits), as we will be using this data for the new Prepayment service call.

## How to test:

Breakpoint in the paymentData we send to the processor, and check that _firstSixDigits_ in _PXToken_ is not empty.

## Prints for layout changes:

None needed.

## Extras:
